hi-ml-cpath/src/health_cpath/configs/classification/BaseMIL.py

```diff
@@ -101,7 +109,11 @@ class BaseMIL(LightningContainer, LoadingParams, EncoderParams, PoolingParams, C
     )
 
     def __init__(self, **kwargs: Any) -> None:
+        """
+        :param eval_num_classes: Number of classes to use when running in eval mode. This is provided because
```

Comment:
What exactly is the problem with having fewer classes in the eval data? If anything really breaks, could it be better to fix that instead?

Also, is this even a issue for binary tasks? Recall that our pipeline treats `num_classes=1` as binary and `=2` as multi-class with 2 categories—so AFAIK we can't even say there's a single class present...?

Reply:
Having fewer classes in eval is not the problem we are solving here. We don't want to rely on the training data to
determine the number of classes, because it means making the training dataset available.